.github/workflows/vfd.yml

```diff
@@ -0,0 +1,47 @@
+name: hdf5 VFD CI
+
+# Run VFD CI daily at 07:00 CDT (12:00 UTC)
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 12 * * *"
+
+permissions:
+  contents: read
+
+jobs:
+  build_and_test:
+    strategy:
+      matrix:
+        build_sys: ["CMake", "Autotools"]
+        build_mode: ["Release", "production", "Debug", "debug"]
+        include:
+          - build_sys: "CMake"
+            build_mode: "Release"
+          - build_sys: "CMake"
+            build_mode: "Debug"
+          - build_sys: "Autotools"
+            build_mode: "production"
+          - build_sys: "Autotools"
+            build_mode: "debug"
+        # Exclude mismatched configurations
+        exclude:
```

Comment:
Seems to need both the include and exclude here, otherwise GitHub either excludes all the configurations or includes all the configurations

Reply:
You could just split this into autotools and cmake. It may be better for clarity.